Documentation/Storage-Configuration/Advanced/ceph-mon-health.md

```diff
@@ -124,3 +124,43 @@ CephCluster CR:
 - `spec.network.hostNetwork`: When enabled or disabled, Rook fails over all monitors, configuring them to enable or disable host networking.
 - `spec.network.Provider` : When updated from being empty to "host", Rook fails over all monitors, configuring them to enable or disable host networking.
 - `spec.network.multiClusterService`: When enabled or disabled, Rook fails over all monitors, configuring them to start (or stop) using service IPs compatible with the multi-cluster service.
+
+## External Monitors
+
+!!! attention
+    This feature is experimental.
+
+It is possible to have both Rook-managed and external monitors in the same Rook cluster.
+One use case for this is 2 datacenter aka 2-AZ (Availability Zone) setup. For 2-AZ setup, Zone outage will lead to loss of k8s control plane and a half of worker nodes hosting Rook mons.
+In this case remaining half of Rook cluster will not be able to form quorum and will be in a stuck state even if half of worker nodes are still up.
+To avoid this situation, external mons can be used to form quorum and keep the cluster running.
+
+If you have external monitors that are not managed by Rook, you have to inform Rook about them, otherwise Rook will remove unknown mons from the quorum. This is done by setting the `mon.externalMonIDs` field in the CephCluster CR. Internal `mon.count` ignores a number of `mon.externalMonIDs`. In other words, if `mon.count = 2`, Rook will spawn 2 internal mons no matter how many external mons are in the cluster and what is their health state. External monitors are supported only for local Rook cluster running in normal mode, meaning that `mon.externalMonIDs` will be ignored for external cluster (`spec.external.enabled: true`) or for `stretchedCluster`.
```

Comment:
In the docs it is preferred to avoid "you", and rather speak in third person.
```suggestion
If there are external monitors, Rook must be aware of them, otherwise Rook will remove the unknown mons from the quorum. This is done by setting the `mon.externalMonIDs` field in the CephCluster CR. The `mon.count` ignores the number of `mon.externalMonIDs`. For example, if `mon.count = 2`, Rook will create two internal mons no matter how many external mons are in the cluster and no matter what their health state might be. External monitors are supported only for the local Rook cluster running in normal mode. The external mons will be ignored for external clusters and stretch clusters.
```